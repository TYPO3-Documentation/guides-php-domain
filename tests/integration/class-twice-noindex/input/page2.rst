=================================
class listed twice throws warning
=================================

..  php:class:: TYPO3\CMS\Core\Test
    :noindex:

    Lorem Ipsum Dolor!
