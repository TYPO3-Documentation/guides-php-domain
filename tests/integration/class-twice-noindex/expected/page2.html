<!-- content start -->
    <div class="section" id="class-listed-twice-throws-warning">
            <h1>class listed twice throws warning</h1>
            <dl class="php class">
    <dt class="sig sig-object php">
        <em class="property"><span class="pre">class</span> </em>
        <span class="sig-name descname">
    <span class="pre fqn" data-namespace="\TYPO3\CMS\Core" title="\TYPO3\CMS\Core\Test">
        Test
    </span>
</span>    </dt>
    <dd>
                    <dl class="field-list simple">
                                    <dt class="field-even">Fully qualified name</dt>
                    <dd class="field-even">
                                                <span class="pre">\TYPO3\CMS\Core\TYPO3\CMS\Core</span>                    </dd>
                </dl>

    <p>Lorem Ipsum Dolor!</p>

    </dd>
</dl>
    </div>
<!-- content end -->
